sql/CMakeLists.txt

```diff
@@ -125,10 +125,18 @@ function(cat_files SRC_FILE_LIST OUTPUT_FILE)
 endfunction()
 
 # Generate the extension file used with CREATE EXTENSION
-cat_files(
-  "${PRE_INSTALL_SOURCE_FILES_VERSIONED};${SOURCE_FILES_VERSIONED};${IMMUTABLE_API_SOURCE_FILES_VERSIONED};${NOTICE_FILE}"
-  ${CMAKE_CURRENT_BINARY_DIR}/${INSTALL_FILE}
-)
+# If in debug mode, don't compile in the NOTICE, so that it doesn't appear in tests
```

Comment:
I'm not sure this is the right fix -- this means I can't run the tests and expect them to pass in Release mode (which I do not irregularly). Do we have a GUC to suppress the NOTICE? That might be a better way of dealing with this -- meaning we can disable the NOTICE when running tests (or the user can so that it doesn't pollute their logs).

Reply:
Really don't want to use a GUC here since it's not really a property of the database. How do you feel about adding an additional make flag that is passed to bootstrap? Then users can also compile without NOTICE.

Reply:
I suppose that works, but I'm definitely going to forget :) It just seems like that kind of thing shouldn't be in the tests.

Maybe for now we just change the logging level wherever it may cause this problem?

Reply:
Ugh ok, that is not entirely sustainable and a little annoying because every new test needs to add it. But it's easy enough to do now.